scanpy/tests/test_normalization.py

```diff
@@ -26,21 +26,21 @@
 X_frac = np.array([[1, 0, 1], [3, 0, 1], [5, 6, 1]])
 
 
-@pytest.mark.parametrize("array_type", ARRAY_TYPES_SUPPORTED)
+@pytest.mark.parametrize("array_type", ARRAY_TYPES)
 @pytest.mark.parametrize("dtype", ["float32", "int64"])
 def test_normalize_total(array_type, dtype):
     adata = AnnData(array_type(X_total).astype(dtype))
     sc.pp.normalize_total(adata, key_added="n_counts")
-    assert np.allclose(np.ravel(adata.X.sum(axis=1)), [3.0, 3.0, 3.0])
+    assert np.allclose(np.ravel(sc._utils.axis_sum(adata.X, axis=1)), [3.0, 3.0, 3.0])
     sc.pp.normalize_total(adata, target_sum=1, key_added="n_counts2")
-    assert np.allclose(np.ravel(adata.X.sum(axis=1)), [1.0, 1.0, 1.0])
+    assert np.allclose(np.ravel(sc._utils.axis_sum(adata.X, axis=1)), [1.0, 1.0, 1.0])
 
     adata = AnnData(array_type(X_frac).astype(dtype))
     sc.pp.normalize_total(adata, exclude_highly_expressed=True, max_fraction=0.7)
-    assert np.allclose(np.ravel(adata.X[:, 1:3].sum(axis=1)), [1.0, 1.0, 1.0])
+    assert np.allclose(np.ravel(sc._utils.axis_sum(adata.X, axis=1))[:2], [2.0, 4.0])
```

Comment:
COO problem mentioned elsewhere forced this change. Can't index into a COO array.